### PR TITLE
fixed typo in elasticsearch dockerfile

### PR DESCRIPTION
**Pull request by monotek**
**What this PR does / why we need it**:
* fixed typo in elasticsearch dockerfile

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes https://github.com/kubernetes/kubernetes/pull/67484#issuecomment-414177808

**Special notes for your reviewer**:

**Release note**:

```release-note
fixed typo in elasticsearch dockerfile
```

### CODE REVIEW

File: cluster/addons/fluentd-elasticsearch/es-image/Dockerfile [hunk 1]
```diff
@@ -17,7 +17,7 @@ FROM docker.elastic.co/elasticsearch/elasticsearch-oss:6.3.2
 VOLUME ["/data"]
 EXPOSE 9200 9300
 
-COPY elasticsearch_logging_discovery run.sh bin/
+COPY elasticsearch_logging_discovery.go run.sh bin/
```
Comment (coffeepac) [inline cluster/addons/fluentd-elasticsearch/es-image/Dockerfile]: we don't want to copy the source file.  we want to copy in the built elasticsearh_logging_discovery binary that gets built in the Makefile from the `binary` directive.

**Review by coffeepac — CHANGES_REQUESTED** [review]
@monotek is there a specific error that you're seeing for this?

File: cluster/addons/fluentd-elasticsearch/es-image/Dockerfile (hunk 1 above)
Comment (monotek) [reply cluster/addons/fluentd-elasticsearch/es-image/Dockerfile]: ah... sorry, my fault...

**Pull request closed without merging**